corehq/apps/app_manager/suite_xml/features/mobile_ucr.py

```diff
@@ -371,7 +374,7 @@ def _get_word_eval(word_translations, default_value):
         title=Text(
             locale=Locale(id=id_strings.report_data_table()),
         ),
-        fields=[_column_to_field(c) for c in config.report(domain).report_columns]
+        fields=[_column_to_field(c) for c in config.report(domain).report_columns if c.type != 'expanded']
```

Comment:
this just displays an empty data column on mobile v1, but it hard failed (xpath error) on v2. It didn't seem to be of much use in v1, so I am just removing it from the table